csharp/src/ProtocolBuffers.Test/CodedInputStreamTest.cs

```diff
@@ -232,66 +230,26 @@ public void DecodeZigZag64()
             Assert.AreEqual(0x7FFFFFFFFFFFFFFFL, CodedInputStream.DecodeZigZag64(0xFFFFFFFFFFFFFFFEL));
             Assert.AreEqual(unchecked((long) 0x8000000000000000L), CodedInputStream.DecodeZigZag64(0xFFFFFFFFFFFFFFFFL));
         }
-        /*
+        
         [Test]
-        public void ReadWholeMessage()
+        public void ReadWholeMessage_VaryingBlockSizes()
         {
-            TestAllTypes message = TestUtil.GetAllSet();
+            TestAllTypes message = GeneratedMessageTest.GetSampleMessage();

```

Comment:
I am not huge fan of using GeneratedMessageTest.GetSampleMessage() from CodedInputStreamTest. There should be a separate class if we need to call GetSampleMessage from multiple classes. The name could be more specific than just TestUtil.